package.json

```diff
@@ -140,22 +147,29 @@
     "@web/test-runner-playwright": "^0.11.0",
     "chokidar-cli": "^3.0.0",
     "comment-parser": "^1.4.1",
+    "convert-source-map": "^2.0.0",
     "custom-elements-manifest": "^2.1.0",
     "esbuild": "^0.25.0",
     "eslint": "^9.31.0",
     "eslint-config-prettier": "^10.1.5",
     "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-lit": "^1.15.0",
     "eslint-plugin-lit-a11y": "^4.1.4",
+    "eslint-plugin-playwright": "^2.2.2",
     "eslint-plugin-sort-class-members": "^1.21.0",
     "eslint-plugin-unicorn": "^60.0.0",
     "globals": "^15.13.0",
     "globby": "^14.0.2",
+    "http-server": "^14.1.1",
```

Comment:
`http-server` is back! 

Playwright is able to serve our coverage report. But it would open a browser tab with the report every time this script is run. If you're okay with that, let me know. I'll replace `http-server` with the equivalent Playwright command.

Reply:
- Storybook: `:6006`
- Playwright: `:6007`
- Coverage Report: `:6008`